_raml/apps/tax-model/examples/get_taxmodel_response.json

```diff
@@ -0,0 +1,4 @@
+{
+  "model": "gross",
+  "displayTaxes": false
```

Comment:
use this as example:

"model": "gross",
 "displayTaxes": yes

...because this is probably the setting you will find in most of the shops